pyqt/pyqt5.json

```diff
@@ -0,0 +1,115 @@
+{
+    "name": "pyqt5",
+    "config-opts": [
+        "--concatenate=1",
+        "--confirm-license",
+        "--enable=QtCore",
+        "--enable=QtDBus",
+        "--enable=QtGui",
+        "--enable=QtNetwork",
+        "--enable=QtOpenGL",
+        "--enable=QtPrintSupport",
+        "--enable=QtQml",
+        "--enable=QtQuick",
+        "--enable=QtSql",
+        "--enable=QtTest",
+        "--enable=QtWebChannel",
+        "--enable=QtWidgets",
+        "--no-designer-plugin",
+        "--no-docstrings",
+        "--no-make",
+        "--no-qml-plugin",
+        "--no-tools",
+        "--api-dir=/app/share/qt/qsci/api/python",
```

Comment:
Is this line really necessary ?

Reply:
I actually can't remember. I've done the sip6 conversion a couple of months back, and left it in WIP state until got back to it today.  
I'll retest this.

Reply:
According to the [doc](https://www.riverbankcomputing.com/static/Docs/sip/command_line_tools.html#cmdoption-sip-build-api-dir), it's useless in our case. You can just drop it.

Reply:
What about apps that depend on QScintilla? Without this `PyQt5.api` will not be installed.

Reply:
1. You're cleaning it on https://github.com/flathub/shared-modules/pull/145/files#diff-e0d667a45d00fde2ccbdd1f55b4b238ea7836804c284e4b89c20fc03a9c7f8afR67 anyway.
2. I don't think most apps will need it. We're back to why I think this is not a good idea to have PyQt as a shared-modules.

Reply:
> You're cleaning it on...

The logic was that it's only needed when packaging an app, not for running.

> We're back to why I think this is not a good idea to have PyQt as a shared-modules.

The fact that you're here pointing out issues and helping improve the packaging of this module, illustrates the importance of having this as a shared module that is more discoverable than a module that hides in some random app packaging.